### ISSUE TITLE
bug: ecctl ignores EC_API_KEY environment variable

### ISSUE THREAD

**Issue by yokawasa** [elastic/ecctl#376]
## Question

ecctl version: 1.0.0

Q: How to skip ecctl init in executing ecctl command.  

My understanding is that I need to configure ecctl with `ecctl init` or set  environment variables like EC_API_KEY or 
 EC_USER & EC_PASS before I execute ecctl command like: 
`ecctl deployment show DEPLOYMENT_ID --kind elasticsearch --plans --host https://api.elastic-cloud.com`

What I've observed is that  even if I've set EC_API_KEY enviroment variable, i still come up with the following message in running `ecctl deployment show` like above.  Is this an expected behavior?  What I'd like to achieve is to run the ecctl command without configuring it with `ecctl init`  but with setting some enviroment variables like `EC_API_KEY`, etc. 

```
missing ecctl config file, please use the "ecctl init" command to initialize ecctl: 1 error occurred:
        * invalid configuration options specified: api_key or user and pass must be specified
```

NOTE:  in ecctl-1.0.0-beta3, I am able to run the command above as long as I set `EC_APIKEY` (NOT EC_API_KEY) environment variable.

**Comment by marclop** [comment]
@yokawasa thanks for raising it. It seems that you have discovered a bug that's present on the released `v1.0.0` version. I've identified the bug and it'll be fixed in the upcoming `v1.0.1` version.

**Comment by marclop** [comment]
Resolved by #381 . It will be available in the upcoming `v1.0.1` release

**Comment by yokawasa** [comment]
@marclop  thank you for the prompt resultion!
